addon/components/connect.js

```diff
@@ -37,9 +38,15 @@ export default (stateToComputed, dispatchToActions=() => ({})) => {
           });
         }
 
-        this.actions = Object.assign({},
-          this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
-        );
+        if (typeof dispatchToActions === 'function') {
+          this.actions = Object.assign({},
+            this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
+          );
+        }
+
+        if (typeof dispatchToActions === 'object') {
+          this.actions = Object.assign({}, this.actions, bindActionCreators.call(this, dispatchToActions, redux.dispatch.bind(redux)));
```

Comment:
I think this also means that actions bound in this way don't get the `this` context, so they can't do things like `this.get` in their functions, which was discussed briefly in https://github.com/ember-redux/ember-redux/pull/60.